pkg/addons/data/ConfigMap/openshift-azure-monitoring/metrics-bridge.yaml

```diff
@@ -4,14 +4,28 @@ data:
     prometheusFederateEndpoint: https://prometheus-k8s.openshift-monitoring.svc:9091/federate
     statsdSocket: /var/etw/mdm_statsd.socket
     series:
-    - kube_node_info
-    - kube_node_status_allocatable_cpu_cores
-    - kube_node_status_allocatable_memory_bytes
-    - kube_node_status_condition
-    - kube_pod_container_status_restarts_total
-    - kube_pod_info
-    - kube_pod_status_phase
-    - kube_pod_status_ready
+    - '{__name__=~"container_.+",namespace=~"(^default$)|(^openshift$)|(^openshift-.+)|(^kube-.+)|(^$)"}'
```

Comment:
extra caret needed on all of these? - `__name__=~"^container_"` ...

Reply:
(nit: can also do `"(^openshift-)|(^kube-)"`)

Reply:
as in - the trailing `.+` is only really needed if it's `.+$`

Reply:
curling without .+ in __name__ didn't return any results (or returned exact match if there was a metric with the exact name, I'm guessing $ is implicit. Not sure if ^ is implicit too, so left it for now in namespaces to avoid too many matches.

Reply:
according to https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors, "regex-matches are fully anchored".  TIL!
so: `'{__name__=~"container_.+",namespace=~"default|openshift|openshift-.+|kube-.+|"}'` ?

Reply:
corrected